Makefile

```diff
@@ -1,13 +1,27 @@
-RM_TOOLS_REPO_URL = git@github.com:ONSdigital/rm-tools.git
+RM_TOOLS_REPO_URL = https://github.com/ONSdigital/rm-tools.git
+RAS_RM_REPO_URL = https://github.com/ONSdigital/ras-rm-docker-dev.git
 
 .PHONY: test system_tests acceptance_tests
 
 install:
 	pipenv install --dev
 
+start_services:
+	git clone --depth 1 ${RAS_RM_REPO_URL} tmp_ras_rm_docker_dev
+	cd tmp_ras_rm_docker_dev\
+	&& make up\
+	&& docker stop oauth2-service\
```

Comment:
These next two lines (oauth2-service) can be removed with the changes to the docker Makefile now in master.